### DISCUSSION TITLE
第3节课作业(23班)

### DISCUSSION THREAD

**Discussion by wangyijunlyy** [InternLM/Tutorial#421] [作业]
作业提交

**Comment by i-square** [comment]
https://i-square.github.io/p/InternLM-tutorial-campsection3-Build-Knowledge-Base-Using-InternLM-and-LangChain/

**Comment by yaoyaotaba** [comment]
https://github.com/yaoyaotaba/-/wiki/%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%AE%9E%E6%88%98%E8%90%A5%EF%BC%88%E7%AC%AC3%E8%AF%BE%E4%BD%9C%E4%B8%9A%EF%BC%89

**Comment by PlateAss** [comment]
https://plateass.github.io/work3

**Comment by Epi10gue** [comment]
http://t.csdnimg.cn/c0LyW

**Comment by YangZhC** [comment]
https://blog.csdn.net/donotpython/article/details/135761564

**Comment by HuJijin** [comment]
https://polished-collision-b6f.notion.site/14ca8254b5a947b5bd70c9b08e82efd2?pvs=4
